File: Logitude/JustWebFreight/WebFreight.Web/DataContracts/DataProviderField.cs
using System.Collections.Generic;

namespace WebFreight.Web.DataContracts
{
    public class DataProviderField
    {
        public DataProviderField()
        {

        }

        public string Name { get; set; }

        public string Text { get; set; }

        public string Type { get; set; }

        public string Expression { get; set; }

        public bool IsChecked { get; set; }

        public int Sort { get; set; }

        public List<DataProviderField> Fields { get; set; }
        public string Translation { get; set; }
    }
}

File: Logitude/AngularModules/AngularModules/Common/DataContracts/DataProviderField.ts
export class DataProviderField {
    public Name: string;
    public Text: string;
    public Type: string;
    public Expression: string;
    public IsChecked: boolean;
    public Sort: number;
    public ClassName: string = "ListBoxItem";
    public FieldsOpened: boolean;
    public Fields: DataProviderField[];
    public DivSelectBackgroud: string;
}
